.github/workflows/ci.yml

```diff
@@ -77,12 +84,22 @@ jobs:
     - name: Run pana analyzer
       run: |
         ./scripts/score_analyzer.sh
+    - name: Set up android enviroment
+      run: |
+        ./scripts/macos_setup_android_env.sh
+    - name: Configure sdkmanager
+      run: |
+        ./scripts/macos_configure-sdkmanager.sh
+    - name: Run android integration tests
+      run: |
+        ./scripts/run-mocked-integration-tests-android.sh
     - name: Run unmocked integration tests
       run: |
         ./scripts/run-unmocked-integration-tests-ios.rb
     - name: Run mocked integration tests
       run: |
         ./scripts/run-mocked-integration-tests-ios.rb
+
     - name: Analyze
       run: |
-        flutter analyze
+        flutter analyze
```

Comment:
I was wondering, why do you delete the last new line?

Reply:
my mistake

Reply:
revert changes